public/i18n/en.json

```diff
@@ -9,7 +9,11 @@
     "message-documentation": "qqq"
   },
   "ftm-ppe-cta": "Let's get our PPE into the hands of those who need it!",
+  "makers-ppe-cta": "Make PPE & get it into the hands of those who need it.",
   "ftm-requester-near-me": "Search for a requester near you. Not sure how to give? <a class=\"text-light\" href=\"javascript:;\" onclick=\"window.location='faq'+window.location.search;\">Read our FAQ</a>.",
+  "maker-request-near-me": "Coordinate with a Maker Group near you. Questions? <a class=\"text-light\" href=\"javascript:;\" onclick=\"window.location='faq'+window.location.search;\">Read our Maker FAQ.</a>.",
+  "ftm-CTA-link-back": "Want to find requesting institutions near you? <a class=\"text-light" href=\"www.findthemasks.com\">Check out Find the Masks.</a>.",
```

Comment:
you're missing the "escape" character on the quote after text-light on both of these, that's why it's red (but also likely causing lots of other issues).

i.e. it needs to be `class=\"text-light\"`

Reply:
doh!